pkgs/development/compilers/gerbil/gerbil-support.nix

```diff
@@ -13,21 +17,25 @@ rec {
 
   # Use this function to create a Gerbil library. See gerbil-utils as an example.
   gerbilPackage = {
-    pname, version, src, meta, package,
-    git-version ? "", version-path ? "config/version.ss",
+    pname, version, src, meta, gerbil-package,
+    git-version ? "", version-path ? "",
     gerbil ? pkgs.gerbil-unstable,
     gambit-params ? pkgs.gambit-support.stable-params,
     gerbilInputs ? [],
     buildInputs ? [],
-    softwareName ? "" } :
+    softwareName ? ""} :
     let buildInputs_ = buildInputs; in
     gccStdenv.mkDerivation rec {
-      inherit src meta pname version;
+      inherit src meta pname version gerbil-package version-path;
```

Comment:
Why change the inherit here?

Reply:
I'm now querying the gerbil-package and version-path attributes from the packages that depend on you, so create a version file with the correct dependencies on the other version files.

Reply:
Maybe it should be in `passthru` then? No difference for downstream use, but names with `-` in direct `mkDerivation` arguments (that tries to export things as environment variables during the build) look suspicious.

Reply:
I didn't know about `passthru`. Maybe `gerbil-package` is it, but `version-path` isn't. I'll have to evaluate each variable...

Reply:
Why `version-path` isn't it? You use is via Nix string variable expansion, not via shell environment anyway.

Reply:
(checked: it gets into `.drv` file but not into the actual environment, the most fragile combination, I guess…)

Reply:
Let's discuss it on IRC some time this week (or next week?)

Reply:
OK, I'm now using `passthru` for `gerbil-package` and `version-path`.